index.js

```diff
@@ -24,7 +24,10 @@ exports.debug_mode = /\bredis\b/i.test(process.env.NODE_DEBUG);
 
 // hiredis might not be installed
 try {
-    parsers.push(require("./lib/parsers/hiredis"));
+    // Confuse bundlers so that they won't try
+    // to include this native module.
+    var hiredisParserPath = "./lib/parsers/hiredis";
+    parsers.push(require(hiredisParserPath));
```

Comment:
It's just browserify :sweat_smile: :sweat_smile: :sweat_smile: 
Adding that code does fix it. but if that somehow hinders readability and you prefer not to add that, I'll work around this issue using browserify's own api.